lib/Comment/styles.scss

```diff
@@ -0,0 +1,56 @@
+.conversation__text__cutoff {
+  &:after {
+    @apply absolute h-8 w-full;
+    content: ' ';
+    left: 40px;
+    top: 96px;
+    background: linear-gradient(
+      to bottom,
+      rgba(255, 255, 255, 0) 0%,
+      rgba(253, 253, 253, 0.5) 50%,
+      #fff 100%
+    );
+  }
+}
```

Comment:
This is a pseudo element that blurs out the bottom of a comments text if it's too long.

Tailwind seems to get a bit weird when you want to position things absolutely, it doesn't have a lot of values for `top, bottom, left, right`. 

It also doesn't seem to have any functionality for `linear-gradient`!

Reply:
Looking at the tailwind docs, they're suggesting that for absolute positioning, you stick an item to the top/left/bottom/right of whatever parent element you want, and then use padding or margins instead of `top/left/bottom/right` values.

> Use the .{top|right|bottom|left|inset}-0 utilities to anchor absolutely positioned elements against any of the edges of the nearest positioned parent.
>
>Combined with Tailwind's padding and margin utilities, you'll probably find that these are all you need to precisely control absolutely positioned elements.

I think we should follow this pattern - we are using tailwind and we should follow it's rules almost religiously, it's great to have those patterns decided for you.

As for the gradient, we can either leave the SCSS file, or go back to design and say that we can't do a linear gradient as it's a technical limitation.

Both I think are fine options, redesigning the comments at this point is definitely out of scope 🔭

Reply:
Depends how far you want to go, it looks like we can extend the values for `top, bottom, left, right` in the theme, maybe we could have them match our new rem spacing values.

Gradient wise it looks like theres a couple of plug ins that people have made to add this in: https://www.npmjs.com/package/tailwindcss-gradients https://github.com/lorisleiva/tailwindcss-plugins/tree/master/gradients 

Both of these are totally optional and only worth it if you can foresee them being useful for other things too.

Reply:
> Looking at the tailwind docs, they're suggesting that for absolute positioning, you stick an item to the top/left/bottom/right of whatever parent element you want, and then use padding or margins instead of `top/left/bottom/right` values.

I read that same paragraph and completely misunderstood what it was saying. I will do this!